pulp_ansible/app/models.py

```diff
@@ -256,6 +256,8 @@ class AnsibleRepository(Repository):
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
+        permissions = (("modify_ansible_repo_content", "Modify Ansible Repository Content"),)
```

Comment:
I didn't realize it when I susggested this name, but there is a naming conventions Permissions use. As is this reads in the list like this:

```
 'Can add ansible repository',
 'Can change ansible repository',
 'Can delete ansible repository',
 'Modify Ansible Repository Content',
 'Can view ansible repository',
 'Can add collection',
 'Can change collection',
 'Can delete collection',
```

So I think the name should be `"Can modify ansible repository content"`.

Reply:
For the "codename" (the slug version) that list currently reads like:

```
 'change_ansiblerepository',
 'delete_ansiblerepository',
 'modify_ansible_repo_content',
 'view_ansiblerepository',
 'add_collection',
 'change_collection',
 'delete_collection',
```

I think what we have right now is good.